device-identifier/requirements.txt

```diff
@@ -8,4 +8,7 @@ pyfiglet==0.8.post1
 loguru==0.6.0
 influxdb-client==1.30.0
 pyinstaller==5.1
-pandas==1.4.3
\ No newline at end of file
+pandas==1.4.3
+shbdeviceidentifier>=0.0.0
+requests>=2.28.1
+influxdb>=5.3.1
```

Comment:
influxdb will have to be removed in future fixes.

Reply:
Why? We are using `influxdb.InfluxDBClient()`.

Reply:
Yes, but that is the old client (as you mentioned :D), so it should be removed. Not necessary now, just a reminder really. I'm resolving this for now.